content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -294,7 +295,7 @@ Server:  Message Received
 ### Procedure of How to run the two programs if using Intellij or any other IDE
```

Comment:
Change:

>### Procedure of How to run the two programs if using Intellij or any other IDE

To:

>### How our application
>#### Procedure of how to run the two programs if using Intellij or any other IDE

And:

>### If using Command prompt/Terminal

To:

>#### If using Command prompt/Terminal

I think **Procedure of how to run the two programs if using Intellij or any other IDE** and **If using Command prompt/Terminal** should be sub-headings of a section about running the application